### PR TITLE
Clarify Tutorial Instructions

### PR DESCRIPTION
**Pull request by pajkosmi**
## Proposed changes

This PR proposes improvements for the User Tutorials---related to discussion in #4193.  The specific changes are listed below.  
### Upgrade instructions

N/A

### Code review checklist

- [x] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [x] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### Further comments

Specifically three changes are addressed:

1. For the Hitchhiker's Guide, I add a line clarifying where to add a filter when visualizing BBH coordinates,
2. For Running & Visualizing, I place a reminder to navigate to the build directory when running `make EvolveScalarWavePlaneWave3D`,
3. For Running CCE, I mention the worldtube hdf5 files need to be obtained from an external collaborator.

### CODE REVIEW

File: docs/Tutorials/BeginnersTutorial.md [hunk 1]
```diff
@@ -187,7 +187,7 @@ depending on what version of `Paraview` you have.)
 \image html beginners_paraview_top.png "Paraview top-bar"
 
 Now you'll have a solid sphere with highlighted lines. To view the interior of
-the domain, you'll need to add a filter. Probably the two most helpful filters
+the domain, you'll need to add a filter. To access the filters, navigate to `Filters` on the top menu bar, hover over `Alphabetical`, and search for your filter of choice.  Probably the two most helpful filters
```
Comment (knelli2) [inline docs/Tutorials/BeginnersTutorial.md]: This line is too long. For our formatting style, we keep lines 80 characters or less. You can either change this by hand, or , if you use VSCode, there are many extensions that will detect and auto-wrap lines longer than 80 characters.

File: docs/Tutorials/CCE.md [hunk 2]
```diff
@@ -76,6 +76,7 @@ CCE using external data are:
         for attribute in input_h5[dset].attrs.keys():
             output_h5[dset].attrs[attribute] = input_h5[dset].attrs[attribute]
   ```
+- Note, these `*.h5` files required as an `input_file`---and also needed in the `.yaml` file---will need to be obtained from an SXS member who has completed a relevant `SpEC` run.
```
Comment (knelli2) [inline docs/Tutorials/CCE.md]: Line is too long. Also use (parentheses) instead of ---dashes---

File: docs/Tutorials/Visualization.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ SpECTRE source files for evolution executables are located in
 `make EXECUTABLE_NAME` where `EXECUTABLE_NAME` is the name of the executable
 as defined in the `CMakeLists.txt` file located in the same directory as the
 source file. For example, to compile the executable that evolves a scalar wave
-using a three-dimensional domain, one runs the command:
+using a three-dimensional domain, navigate to `/spectre/build/`, then one runs the command:
```
Comment (knelli2) [inline docs/Tutorials/Visualization.md]: Replace `/spectre` with `$SPECTRE_ROOT` or `$SPECTRE_HOME`. That way people don't try to look in the root directory for spectre. We aren't consistent in the docs on when we use ROOT vs HOME so you can choose whichever you prefer. Also I think this line is also too long.

**Review by knelli2 — CHANGES_REQUESTED** [review]
Thanks for noting these changes as you went through the tutorials! Just a few things which you can squash directly.

Just for future reference, we have a few webpages for PRs and code review. One on [contributing to SpECTRE](https://spectre-code.org/contributing_to_spectre.html) and one on [code review guidelines](https://spectre-code.org/code_review_guide.htmll).

**Review by knelli2 — COMMENTED** [review]
Looks good! Since there aren't a lot of changes, you can squash these all into one commit and just explain the changes in the long commit message.

File: docs/Tutorials/BeginnersTutorial.md [hunk 4]
```diff
@@ -223,7 +223,9 @@ depending on what version of `Paraview` you have.)
 \image html beginners_paraview_top.png "Paraview top-bar"
```
Comment (knelli2) [inline docs/Tutorials/BeginnersTutorial.md:224]: Commit message: You still have the fixup commit message in there. Could you remove that?

Comment (pajkosmi) [reply docs/Tutorials/BeginnersTutorial.md:224]: Sure thing.

**Review by knelli2 — APPROVED** [review]

**Review by nilsdeppe — APPROVED** [review]
LGTM! Thanks for the improvements!

image

**Pull request merged**